src/pages/og-image/players/[id]-[nickname].png.ts

```diff
@@ -0,0 +1,157 @@
+                    ${highestLeague.league} ${highestLeague.tier}
+                  </p>
+                  <h1 class="w-full text-gray-200 text-4xl capitalize m-0 pb-3">${player.nickname}</h1>
+                </div>
+              </div>
+              <div class="flex items-center bg-${color}-800/10 border-l border-${color}-500/50 py-3">
+                <div class="flex flex-col w-1/4 items-center ${highestLeague.tier == 1 ? "ml-4" : ""}">
+                  <img src="${leagueIcon}" width="150" />
+                </div>
+                <div class="flex flex-col w-1/3 ml-8">
+                  <p class="m-0 text-gray-400 text-3xl font-bold min-w-32 mb-1">Rank</p>
+                  <p class="m-0 text-gray-100 text-4xl font-bold"># ${highestLeague.rank}</p>
+                </div>
+                <div class="flex flex-col w-1/3">
+                  <p class="m-0 text-gray-400 text-3xl font-bold mb-1">Points</p>
+                  <p class="m-0 text-gray-100 text-4xl font-bold">${highestLeague?.points?.toFixed()}</p>
+                </div>
+              </div>
+              <div class="flex mt-7 w-full">
+                <div class="flex flex-col pl-8 mr-20">
+                  <p class="m-0 text-gray-400 font-bold text-2xl">MMR</p>
+                  <p class="m-0 text-gray-100 text-3xl font-bold">${highestLeague?.mmr?.toFixed()}</p>
+                </div>
+                <div class="flex flex-col mr-14">
+                  <p class="m-0 text-gray-400 font-bold text-2xl">Win Rate</p>
+                  <p class="m-0 text-gray-100 text-3xl font-bold">${highestLeague?.win_rate?.toFixed()}%</p>
+                </div>
+                <div class="flex flex-col w-60 h-16 mt-2 -ml-5">
+                  <p class="m-0 -mb-3 text-gray-400 font-bold text-3xl flex self-end">
+                    <span style="color:#55EA00;">${highestLeague.wins}</span> 
+                    <span class="mx-2"> - </span>
+                    <span style="color:#EA0000;">${highestLeague.losses}</span>
+                  </p>
+                  <div class="flex h-4 mt-4">
+                    <div class="flex" style="width: ${winRateGreenBar}0%;background-color:#55EA00;" />
+                    <div class="flex" style="width: ${winRateRedBar}0%;background-color:#EA0000;" />
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div 
+               style="background: url('${banner}');background-size: 535px 450px;background-repeat:no-repeat;"
+               class="flex w-full h-full border-l border-${color}-800/50"
+             />
+            </div>
+          </div>
+        </div>
+      `
+    } else {
+      cardHTML += `
+        <div 
+          class="flex w-full rounded-xl items-center justify-center"
+          style="
+            background: url('https://${host}/images/game/vanguard-base-hero.jpg');
+            background-size: 1050px 500px; background-repeat: no-repeat;
+            height: 100%;"
```

Comment:
`height: 100%` -> `h-full`
`background-repeat: no-repeat;` -> `bg-no-repeat`

and you can actually also do `background-size: 1050px 500px` as `bg-[length:10500px_500px]`

Reply:
Thanks. I was able to remove those properties from the banners. But for some reason the bg-[length: 1050px_500px] doesn't seem to be working and I suspect it's satori-html, as it has some quirks.